client/reader/list-manage/index.jsx

```diff
@@ -126,7 +126,13 @@ function ReaderListEdit( props ) {
 										'You can export this list to use on other services. The file will be in OPML format.'
 									) }
 								</p>
-								<ReaderExportButton exportType={ READER_EXPORT_TYPE_LIST } listId={ list.ID } />
+								<ReaderExportButton
+									exportType={ READER_EXPORT_TYPE_LIST }
+									listId={ list.ID }
+									filename={ `reader-list-${ props.slug
+										.replace( /[^a-z0-9]/gi, '_' )
```

Comment:
```suggestion
										.replace( /[^a-z0-9]/gi, '-' )
```

keeps filenames for `Foo Bar` as `reader-list-foo-bar.opml` instead of `reader-list-foo_bar.opml`